### ISSUE TITLE
Auto Hire Engineers not using selected LC

### ISSUE THREAD

**Issue by westlakem** [KSP-RO/RP-1#2368]
I'm clicking on "Auto hire" with LC2 selected
![image](https://github.com/KSP-RO/RP-1/assets/4505074/311f2076-fdd0-4b33-979c-0b5b7bdd5abb)
The menu pops up with LC1
![image](https://github.com/KSP-RO/RP-1/assets/4505074/fc6ff8b3-ce2a-4980-80be-b7b53c51fe7a)

**Comment by siimav** [comment]
Yeah, didn't notice that the LC switch button in that window is decoupled from what the actual active LC is.
